include/wx/qt/datectrl.h

```diff
@@ -0,0 +1,64 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/qt/datectrl.h
+// Purpose:     wxDatePickerCtrl for Qt
+// Author:      Ali Kettab
+// Modified by:
+// Created:     2023-10-12
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_QT_DATECTRL_H_
+#define _WX_QT_DATECTRL_H_
+
+class QDateEdit;
+
+// ----------------------------------------------------------------------------
+// wxDatePickerCtrl
+// ----------------------------------------------------------------------------
+
+class WXDLLIMPEXP_ADV wxDatePickerCtrl : public wxDatePickerCtrlBase
+{
+public:
+    // ctors
+    wxDatePickerCtrl() { }
```

Comment:
Again, not worth changing it here, but in the future, consider writing
```suggestion
    wxDatePickerCtrl() = default;
```
instead.